### DISCUSSION TITLE
How to save simulation?

### DISCUSSION THREAD

**Discussion by okienative99** [niels747/2D-Weather-Sandbox#36] [Q&A] [answered]
Hey everyone! Wanting to know how you properly save a simulation?

**Comment by Meso22** [comment]
There is only one way to save the simulation.

**Comment by MET905** [comment, marked as answer]
![sq](https://user-images.githubusercontent.com/113155608/208591065-588573e2-3047-4ff5-ab3d-89b0805708e0.png)
